core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt

```diff
@@ -163,27 +173,26 @@ fun LpOutlinedTextFieldMail(
         },
         modifier = modifier,
         label = { Text(label, style = MaterialTheme.typography.bodyMedium) },
+        placeholder = { Text(placeholder) },
```

Comment:
please add a gray color to the label and placeholder

Reply:
Hi Meli!, I think I did defined it as gray, you can check it on line 185 if I'm not mistaken, and as for the label the color was already defined on line 181, let me know if I'm mistaken :smiley:

Reply:
Amm nop, you missed the color in the text on line 199, we should use the color tertiary on that as well.